### DISCUSSION TITLE
Error in alpine Linux with matchers

### DISCUSSION THREAD

**Discussion by pablogsal** [ymyzk/tox-gh-actions#152] [Q&A] [answered]
When using the last version of tox-gh-actions on alpine Linux it fails with this error:

```
Error: Unable to process command '::add-matcher::/usr/local/lib/python3.10/site-packages/tox_gh_actions/matcher.json' successfully. 

Error: Could not find a part of the path '/usr/local/lib/python3.10/site-packages/tox_gh_actions/matcher.json'. 

original envconfigs: ['.package', 'py310-cov', 'py311', 'py310', 'py39', 'py38', 'lint']

```

**Comment by ymyzk** [comment]
Hello @pablogsal, thanks for reporting an issue and sorry for my slow response.

This issue looks similar to #113 and #114 but we haven't verified if this works on Alpine Linux. To reproduce the issue in my environment, could you share how you're running Alpine Linux on GitHub Actions with me? Thanks.

**Comment by ymyzk** [comment, marked as answer]
If there are people facing this error, tox-gh-actions v2.12.0 or later provides an option to disable the problem matcher: https://github.com/ymyzk/tox-gh-actions/blob/v2.12.0/README.md#disable-problem-matchers

As it's probably difficult for tox-gh-action to tell whether it can safely enable the matcher or not perfectly. I hope the new option can save users facing this error. Thanks.

**Comment by pablogsal** [comment]
Thanks a lot @ymyzk ❤️
